baseclasses/pyAero_problem.py

```diff
@@ -965,3 +1147,18 @@ def __init__(self, key, value, lower, upper, scale, offset, dvOffset, addToPyOpt
         self.addToPyOpt = addToPyOpt
         self.family = family
         self.units = units
+
+        if isinstance(value, float):
+            self.size = 1
+        elif isinstance(value, numpy.ndarray):
+            self.size = value.size
+        else:
+            size = None
+            # TODO raise an error?
```

Comment:
Makes sense... unless we want to "filter" these inputs before we instantiate the `aeroDV` container.
Are we using this `size` attribute anywhere yet? Definitely useful anyway